I'm still not sure why is it failing, even worse it only happens on my review vm (that is using a few weeks older stripped down config of my main machine).

On the VM, I get these messages for most packages:
```console
gradio-sans-reverse-dependencies>  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @self/app@1.45.0 build: `vite build && node after_build.js --emptyOutDir`
gradio-sans-reverse-dependencies> Exit status 134
gradio-sans-reverse-dependencies>  ELIFECYCLE  Command failed with exit code 134.
```
```console
┃        > Running phase: checkPhase
┃        > --- FAIL: TestHookOnErrorHandling (68.64s)
┃        >     --- FAIL: TestHookOnErrorHandling/retry (4.62s)
┃        >         backresthandler_test.go:568: expected backup operation cancelled status, got STATUS_INPROGRESS
┃        > FAIL
┃        > FAIL github.com/garethgeorge/backrest/internal/api   81.040s
┃        > FAIL
```

However on my main machine there are no new breakages:

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 384815 --skip-package documenso --skip-package follow --skip-package shadcn`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>froide</li>
    <li>froide.dist</li>
    <li>python312Packages.froide</li>
    <li>python312Packages.froide.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>typespec</li>
    <li>wrangler</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 100 packages built:</summary>
  <ul>
    <li>apache-answer</li>
    <li>artalk</li>
    <li>astro-language-server</li>
    <li>autobrr</li>
    <li>autoprefixer</li>
    <li>backrest</li>
    <li>cdxgen</li>
    <li>clash-verge-rev</li>
    <li>daed</li>
    <li>deltachat-desktop</li>
    <li>en-croissant</li>
    <li>equibop</li>
    <li>equicord</li>
    <li>flood</li>
    <li>gitbutler</li>
    <li>gitify</li>
    <li>goofcord</li>
    <li>gui-for-clash</li>
    <li>gui-for-singbox</li>
    <li>heroic</li>
    <li>heroic-unwrapped</li>
    <li>hoarder</li>
    <li>homebox</li>
    <li>homer</li>
    <li>jellyseerr</li>
    <li>legcord</li>
    <li>lemmy-ui</li>
    <li>metacubexd</li>
    <li>misskey</li>
    <li>modrinth-app</li>
    <li>modrinth-app-unwrapped</li>
    <li>moonfire-nvr</li>
    <li>moonlight</li>
    <li>n8n</li>
    <li>ni</li>
    <li>ocis</li>
    <li>overlayed</li>
    <li>parca</li>
    <li>pds</li>
    <li>pgrok</li>
    <li>pgrok.server</li>
    <li>pingvin-share.backend</li>
    <li>piped</li>
    <li>pnpm_9</li>
    <li>podman-desktop</li>
    <li>porn-vault</li>
    <li>postcss-cli</li>
    <li>pot</li>
    <li>prisma</li>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python312Packages.django-filingcabinet</li>
    <li>python312Packages.django-filingcabinet.dist</li>
    <li>python312Packages.gradio</li>
    <li>python312Packages.gradio-client</li>
    <li>python312Packages.gradio-client.dist</li>
    <li>python312Packages.gradio-pdf</li>
    <li>python312Packages.gradio-pdf.dist</li>
    <li>python312Packages.gradio.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>python313Packages.django-filingcabinet</li>
    <li>python313Packages.django-filingcabinet.dist</li>
    <li>python313Packages.gradio</li>
    <li>python313Packages.gradio-client</li>
    <li>python313Packages.gradio-client.dist</li>
    <li>python313Packages.gradio-pdf</li>
    <li>python313Packages.gradio-pdf.dist</li>
    <li>python313Packages.gradio.dist</li>
    <li>renovate</li>
    <li>rmfakecloud</li>
    <li>rquickshare</li>
    <li>rquickshare-legacy</li>
    <li>rsshub</li>
    <li>siyuan</li>
    <li>sketchybar-app-font</li>
    <li>slimevr</li>
    <li>stylelint-lsp</li>
    <li>surrealist</li>
    <li>synchrony</li>
    <li>syncyomi</li>
    <li>tabby-agent</li>
    <li>tailwindcss-language-server</li>
    <li>taler-challenger</li>
    <li>taler-merchant</li>
    <li>taler-sync</li>
    <li>taler-wallet-core</li>
    <li>teleport (teleport_16)</li>
    <li>teleport.client (teleport_16.client)</li>
    <li>teleport_17</li>
    <li>teleport_17.client</li>
    <li>vencord</li>
    <li>vencord-web-extension</li>
    <li>vesktop</li>
    <li>vikunja</li>
    <li>wealthfolio</li>
    <li>webcord-vencord</li>
    <li>zammad</li>
    <li>zenn-cli</li>
    <li>zipline</li>
  </ul>
</details>

--- a/pkgs/development/tools/pnpm/default.nix
+++ b/pkgs/development/tools/pnpm/default.nix
@@ -12,8 +12,8 @@ let
       hash = "sha256-2qJ6C1QbxjUyP/lsLe2ZVGf/n+bWn/ZwIVWKqa2dzDY=";
     };
     "9" = {
-      version = "9.15.5";
-      hash = "sha256-hHIWjD4f0L/yh+aUsFP8y78gV5o/+VJrYzO+q432Wo0=";
+      version = "9.15.7";
+      hash = "sha256-HBjZi6W/BzT8fij2yPG0OxvNhmd33s3igdFIWTkGA/w=";
     };
     "10" = {
       version = "10.4.1";

